mysql-test/main/subselect_mat_analyze_json.result

```diff
@@ -0,0 +1,948 @@
+set @save_optimizer_switch=@@optimizer_switch;
+create table t1 (a int);
+create table t2 (b int);
+insert into t1 values (null), (1), (2), (3);
+insert into t2 values (3), (4);
+set @@optimizer_switch = "materialization=on,in_to_exists=off,semijoin=off";
+explain format=json select * from t1 where a in (select b from t2);
+EXPLAIN
+{
+  "query_block": {
+    "select_id": 1,
+    "table": {
+      "table_name": "t1",
+      "access_type": "ALL",
+      "rows": 4,
+      "filtered": 100,
+      "attached_condition": "<in_optimizer>(t1.a,t1.a in (subquery#2))"
+    },
+    "subqueries": [
+      {
+        "materialization": {
+          "query_block": {
+            "select_id": 2,
+            "table": {
+              "table_name": "t2",
+              "access_type": "ALL",
+              "rows": 2,
+              "filtered": 100
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+# "Complete match" execution strategy
+analyze format=json select * from t1 where a in (select b from t2);
+ANALYZE
+{
+  "query_block": {
+    "select_id": 1,
+    "r_loops": 1,
+    "r_total_time_ms": "REPLACED",
+    "table": {
+      "table_name": "t1",
+      "access_type": "ALL",
+      "r_loops": 1,
+      "rows": 4,
+      "r_rows": 4,
+      "r_table_time_ms": "REPLACED",
+      "r_other_time_ms": "REPLACED",
+      "filtered": 100,
+      "r_filtered": 25,
+      "attached_condition": "<in_optimizer>(t1.a,t1.a in (subquery#2))"
+    },
+    "subqueries": [
+      {
+        "materialization": {
+          "r_strategy": "index_lookup",
+          "r_loops": 3,
+          "r_index_lookup_loops": 3,
```

Comment:
We should not add "_loops" to all counters. 
Please change to "r_index_lookups".

Reply:
Fixed